### DISCUSSION TITLE
在docker中使用无法正常预测出结果

### DISCUSSION THREAD

**Discussion by raymondwm** [openai/whisper#1480] [Q&A] [answered]
<img width="1571" alt="image" src="https://github.com/openai/whisper/assets/110003070/60c2f062-1865-48a0-86b3-08f26eeaa3ed">
如图，会卡在这里。
我试了3.8,3.10都是这样的情况， 模型都是加载成功的。
我在本地直接试了各个版本的，都没有问题。

不管是用代码还是命令行方式执行，本地都是可以，在docker内部就会卡在那里。

**Comment by Tex2002ans** [comment]
I translated your title+post using DeepL.

Chinese->English:

> Using in docker does not predict the results properly #1480
>
> As shown in the picture, it gets stuck here. I tried 3.8,3.10 and this is the case, the models are loaded successfully. I have tried all versions directly locally and there is no problem.
>
> It doesn't matter if it's executed in code or command line, it's fine locally, but inside docker it's stuck there.

Maybe that can help others answer your question.

**Comment by jongwook** [comment, marked as answer]
Thank you for posting the translation! It looks like the CPU is just taking so slow. It's generally way slower to run Whisper on CPU, and it could become extremely slower if you have insufficient system memory and it's paging out to the disk. I'd recommend trying on a GPU node. You'll probably need to use an nvidia docker image and a GPU-enabled version of pytorch if you are already on a GPU node.
